.github/workflows/release-container.yml

```diff
@@ -0,0 +1,47 @@
+  name: Release Keycloak container
+  on:
+    workflow_dispatch:
+    push:
+      tags:
+        - '[0-9]+.[0-9]+.[0-9]+'
+  concurrency: keycloak-container-release
+
+  jobs:
+    publish:
+      if: github.repository == 'keycloak/keycloak'
+      runs-on: ubuntu-latest
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        
+      - name: Container metadata
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: quay.io/keycloak/keycloak
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: quarkus/container
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
```

Comment:
This condition is always `true` do you see any usage of this Job from PRs?

Reply:
This is just a copy/paste from examples, but left it in as the workflow hasn't been tested with PRs so not sure what tags/etc that would result in, nor would it actually work as in main you'd have to build KC first. So this is just left in there as a "safe-guard" in-case someone by mistake changes the "on" stuff.